client/state/utils.js

```diff
@@ -16,13 +16,15 @@ import warn from 'lib/warn';
 /**
  * Module variables
  */
+const memoizedValidator = memoize( validator );
```

Comment:
The idea is to avoid compiling the schema each time. I suppose this doesn't make sense unless we reuse schemas in different reducer trees